internal/log/level.go

```diff
@@ -73,7 +71,7 @@ func (level Level) ColouredString() (s string) {
 	}
 
 	c := color.New(attribute)
-	return c.Sprint(level.String())
+	return c.Sprintf("%-8s", level.String())
```

Comment:
Awesome 👍 Didn't know it would handle it properly that well!

Reply:
Me neither! Was pleasantly suprised at this